IM_calculation/scripts/calculate_ims.py

```diff
@@ -219,9 +251,11 @@ def main():
         im_options["FAS"] = calc.validate_fas_frequency(args.fas_frequency)
 
     # Create output dir
-    utils.setup_dir(args.output_path)
-    if not args.simple_output and args.advanced_ims is None:
-        utils.setup_dir(os.path.join(args.output_path, calc.OUTPUT_SUBFOLDER))
+    if is_master:
+        utils.setup_dir(args.output_path)
+        utils.setup_dir(Path(args.output_path) / "stations")
```

Comment:
If you set the type of "--output_path" to Path, you don't need to do this down here